lib/wallet/send.dart

```diff
@@ -122,6 +117,10 @@ class _SendState extends State<Send> {
                         Row(
                           mainAxisAlignment: MainAxisAlignment.end,
                           children: [
+                            TextButton.icon(
+                                onPressed: () => {GoRouter.of(context).go(QrScan.route)},
+                                icon: const Icon(Icons.qr_code_scanner),
+                                label: const Text("QR Scan")),
                             SubmitButton(
```

Comment:
nit: A little bit of space would be nice here. 

e.g.
const SizedBox(width: 5)

Reply:
done :)